apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -29,6 +29,9 @@ def __init__(self):
         super().__init__()
         self.total_duration = 0  # [s] 0 means infinity
         self.save_partial = True
+        self.save_each_nrmeas = 100
```

Comment:
I think you need to add these new attributes to the `__str__` method.

Reply:
yes, I forgot it! thank you!